config/checker_guideline_map.json

```diff
@@ -69,6 +69,9 @@
     "bugprone-narrowing-conversions": {
       "sei-cert": ["exp45-c", "fio34-c", "flp34-c", "flp36-c"]
     },
+    "bugprone-macro-repeated-side-effects": {
```

Comment:
Don't we keep checker names in alphabetical order in this file? So this one should come before `bugprone-narrowing-conversions` checker.